### PR TITLE
Adding slide tab, animated expand and animated blurred slide items

### PR DESCRIPTION
**Pull request by yunweneric**

### CODE REVIEW

File: lib/main.dart
Comment (ChunhThanhDe) [inline lib/main.dart:1]: This is very necessary! the first time I ran the project, I looked at this Main and it made me confused xD

Comment (yunweneric) [reply lib/main.dart:1]: The main has been removed in the most recent commit. 

Rather the entry of the app is found in lib/app/core

Comment (yunweneric) [reply lib/main.dart:1]: You will find main_dev and main_prod. These are flavors for different environments 


To run the app in vscode just head to your run and debug section then you'll see launch dev and launch prod configuration

Comment (ChunhThanhDe) [reply lib/main.dart:1]: yeah, i see it. xD

**Pull request merged**